site/e2e/playwright.config.ts

```diff
@@ -7,13 +8,33 @@ import {
   e2eFakeExperiment1,
   e2eFakeExperiment2,
   gitAuth,
+  requireTerraformTests,
 } from "./constants";
 
 export const wsEndpoint = process.env.CODER_E2E_WS_ENDPOINT;
 
 // This is where auth cookies are stored!
 export const storageState = path.join(__dirname, ".auth.json");
 
+if (requireTerraformTests) {
+  try {
+    // If running terraform tests, verify the requirements exist in the
+    // environment.
+    //
+    // These execs will throw an error if the status code is non-zero.
+    // So if both these work, then we can launch terraform provisioners.
+    execSync("terraform --version");
+    execSync("docker --version");
+  } catch (e) {
```

Comment:
nit: unused variable (linters have a hard time catching this one because this capture hasn't always been optional)

```suggestion
  } catch {
```